app/src/main/java/com/example/calculator/HistoryActivity.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.calculator
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.graphics.text.LineBreaker.JUSTIFICATION_MODE_INTER_WORD
+import android.os.Build
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_history.*
+import kotlinx.android.synthetic.main.activity_main.*
+import java.lang.StringBuilder
+
+
+@Suppress("PLUGIN_WARNING")
+class HistoryActivity : AppCompatActivity() {
+
+    @SuppressLint("WrongConstant")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_history)
+        val historyClassAction = intent.extras?.getParcelable<HistoryListAction>("historyAction")
+        val historyClassAnswer = intent.extras?.getParcelable<HistoryListAnswer>("historyAnswer")
+        val historylistAnswer = historyClassAnswer?.listAnswer
```

Comment:
why did you create two separate data classes? You can create one that contains two lists: answer and details.